lib/go/framesql/frame.go

```diff
@@ -12,7 +11,7 @@ import (
 
 func EvaluateInFrame(expression string, input *data.Frame) (any, error) {
 	if strings.TrimSpace(expression) == "" {
-		return nil, errors.New(strings.TrimSpace(fmt.Sprintf("empty/invalid expression. %s", expression)))
+		return nil, ErrEmptySummarizeExpression
```

Comment:
Should you also log the expression in case we need to find out the query or this surfaced elsewhere?

Reply:
It should be always just empty string or space as we do a check `strings.TrimSpace(expression) == "" ` . Do you think it would still be valuable, as it is always going to be same/similar thing?